spec/ngsild_swagger.json

```diff
@@ -0,0 +1,1077 @@
+{
+    "swagger": "2.0",
```

Comment:
can we go for 3.0?

Reply:
I need check if JSON can be used to specify swagger 3.0 definitions since when I was working on this just YAML was allowed.